src/streaming/controllers/PlaybackController.js

```diff
@@ -270,7 +271,7 @@ MediaPlayer.dependencies.PlaybackController = function () {
 
             // do nothing if common earliest time has not changed or if the firts segment has not been appended or if current
             // time exceeds the common earliest time
-            if ((currentEarliestTime === commonEarliestTime[id]) || !firstAppended.ready || (time > commonEarliestTime[id])) return;
+            if ((currentEarliestTime === commonEarliestTime[id] && (time === currentEarliestTime)) || !firstAppended.ready || (time > commonEarliestTime[id])) return;
```

Comment:
@KozhinM Can you please briefly describe why this change was required? Just for my curiosity :)
Thanks!